app/lib/features/space/settings/pages/apps_settings_page.dart

```diff
@@ -0,0 +1,487 @@
+                    initialValue: false,
+                    onToggle: (newVal) {},
+                  ),
+                ],
+              ),
+            );
+          }
+
+          return Scaffold(
+            appBar: AppBar(title: const Text('Apps Settings')),
+            body: SettingsList(
+              sections: [
+                SettingsSection(
+                  title: const Text('Active Apps'),
+                  tiles: [
+                    SettingsTile.switchTile(
+                      title: const Text('Updates'),
+                      enabled: canEdit,
+                      description: const Text(
+                        'Post space-wide updates',
+                      ),
+                      initialValue: news.active(),
+                      onToggle: (newVal) async {
+                        final updated = news.updater();
+                        updated.active(newVal);
+                        final builder = appSettings.updateBuilder();
+                        builder.news(updated.build());
+                        await space.updateAppSettings(builder);
+                      },
+                    ),
+                    SettingsTile.switchTile(
+                      title: const Text('Pins'),
+                      enabled: canEdit,
+                      description: const Text(
+                        'Pin important information',
+                      ),
+                      initialValue: pins.active(),
+                      onToggle: (newVal) async {
+                        final updated = pins.updater();
+                        updated.active(newVal);
+                        final builder = appSettings.updateBuilder();
+                        builder.pins(updated.build());
+                        await space.updateAppSettings(builder);
+                      },
+                    ),
+                    SettingsTile.switchTile(
+                      title: const Text('Events Calendar'),
+                      enabled: canEdit,
+                      description: const Text(
+                        'Calender with Events',
+                      ),
+                      initialValue: events.active(),
+                      onToggle: (newVal) async {
+                        final updated = events.updater();
+                        updated.active(newVal);
+                        final builder = appSettings.updateBuilder();
+                        builder.events(updated.build());
+                        await space.updateAppSettings(builder);
+                      },
+                    ),
+                  ],
+                ),
+                ...moreSections,
+              ],
+            ),
+          );
+        },
+        loading: () => const Center(child: Text('loading')),
+        error: (e, s) => Center(
+          child: Text('Error loading app settings: $e'),
+        ),
+      ),
+    );
+  }
+}
+
+class ChangePowerLevel extends StatefulWidget {
```

Comment:
I see this being common widget used. It's all good for the first iteration but we might want to have it reusable for member permissions too.